### PR TITLE
[release] Perform 4.11 release

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

-  4.11 Release

### CODE REVIEW

**Comment by Harshg999** [comment]
Awesome work!
@agl29 - Lets create a doc for capturing the release notes and once finalised, we can populate the content in the release notes here. I feel this can be much easier for collaborating and reviewing the content.

**Comment by Harshg999** [comment]
Curious: I see a `hueversion.py` file in every app/module having a 4.10 version. However, it does not map to any previous commit, is it a generated file?

**Comment by agl29** [comment]
> Curious: I see a `hueversion.py` file in every app/module having a 4.10 version. However, it does not map to any previous commit, is it a generated file?

Not Sure, Need to check on that.

**Comment by Harshg999** [comment]
Hi @pychheda, will it be possible for you to review the upstream 4.11 release notes and blog posts?

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+<a href="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png">
+  <img src="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png" />
+</a>
+
+&nbsp;
+
+**Note: This is the last Hue release supporting Python2.7. From future releases, Hue will support Python3.8+**
+
+
+Here is a summary of the [main improvements](/categories/version-4.11/) of 4.11 on top of the previous [4.10](/blog/hue-4-10-sql-scratchpad-component-rest-api-small-file-importer-slack-app/) release:
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:60]: Here's a summary of [what's new](/categories/version-4.11/) in Hue 4.11:

(Please remove "on top of the previous [4.10](/blog/hue-4-10-sql-scratchpad-component-rest-api-small-file-importer-slack-app/) release" and add the link to 4.10 at the bottom of the page)

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+
+#### Supporting Iceberg Syntax in Hue importer
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:63]: Ability to import and query Iceberg tables from Hue

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+Apache Iceberg is a high-performance format for huge analytic tables. Now you can create a Iceberg Table through Hue importer.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:65]: Apache Iceberg is a high-performance table format and extends multifunction analytics to a petabyte scale for multi-cloud and hybrid use cases. You can now create an Iceberg table using the Hue Importer. 

We've also updated the autocomplete and syntax checker for Hive and Impala to support the latest Iceberg syntax when you write queries.

For more information, see [Creating Iceberg tables in Hue](https://gethue.com/blog/2022-10-11-creating-iceberg-tables-in-hue/) and Iceberg-Hue integration](https://cdn.gethue.com/uploads/2022/10/iceberg1.gif).

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+We’ve also updated the autocomplete and syntax checker for Hive and Impala to support the latest Iceberg syntax.
+
+Read more about the [Creating Iceberg tables in Hue](https://gethue.com/blog/2022-10-11-creating-iceberg-tables-in-hue/).
+
+![Iceberg Hue integration](https://cdn.gethue.com/uploads/2022/10/iceberg1.gif)
+
+
+#### SparkSQL Improvements
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:73]: improvements

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+Hue leverages Apache Livy 3 to support Spark SQL queries in Hue on the Apache Spark 3 engine. Since booting a new Livy session was slow (around 30-45s), we are now caching the session details per user level for faster query experience.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:75]: Hue leverages Apache Livy 3 to support Spark SQL queries in Hue on the Apache Spark 3 engine. Booting a new Livy session was slow, and took around 30-45s. Hue now caches the session details per user for a faster query experience. Caching session details also helps in autocompleting databases, tables, and column names, improving the search experience.
(I don't know what per user level means).

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+Read more about the [Creating Iceberg tables in Hue](https://gethue.com/blog/2022-10-11-creating-iceberg-tables-in-hue/).
+
+![Iceberg Hue integration](https://cdn.gethue.com/uploads/2022/10/iceberg1.gif)
+
+
+#### SparkSQL Improvements
+
+Hue leverages Apache Livy 3 to support Spark SQL queries in Hue on the Apache Spark 3 engine. Since booting a new Livy session was slow (around 30-45s), we are now caching the session details per user level for faster query experience. 
+
+This also helps in database/table/column autocomplete in the left and right assist.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:77]: You can remove this sentence.

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+We also show full table details in the Table Browser. Hue now cleans up unused Livy sessions as well.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:79]: Hue automatically cleans up unused Livy sessions.

(Hue showing full table details in the Table browser looks like a fixed issue. Doesn't quite fit well here.)

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+We now have a dedicated autocomplete and syntax checker available for Spark SQL! It supports all Spark SQL statement types up to date with version 3.3.1.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:81]: Hue has a dedicated autocomplete and syntax checker for Spark SQL. Hue supports all Spark SQL statement types, up-to-date with version 3.3.1.

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+On top of this we’ve also included the UDF library for Spark SQL, accessible in the autocomplete as well as the right assist panel. It covers documentation of the built in functions of Spark SQL 3.3.1.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:83]: We've also integrated the UDF library for Spark SQL in the autocomplete code, as well as in the right assist panel. The inline help includes all built-in functions of Spark SQL 3.3.1.

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+![functioning SparkSQL query editor](https://cdn.gethue.com/uploads/2023/01/sparksql_left_right_assist.gif)
+
+
+#### HPL/SQL Support
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:88]: Added support for HPL/SQL

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+HPL/SQL is an Apache open source procedural extension for SQL for Hive users. It has its own grammar. It is included with Apache Hive from version 2.0. 
+
+You can enable the HPL/SQL dialect via desktop/conf/hue.ini config file section.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:92]: You can enable the HPL/SQL dialect by adding the following lines in the `desktop/conf/hue.ini` configuration file:

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+        [[[hplsql]]]
+        name=Hplsql
+        interface=hiveserver2
+
+Read more about the [HPL/SQL Support](https://gethue.com/blog/2022-02-01-hplsql-support/).
+
+![Example](https://cdn.gethue.com/uploads/2022/02/Hplsql_example1.png)
+
+
+#### Tech stack & Tooling
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:105]: Improvements to the tech stack and tooling

File: docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+
+- [Create SQL tables from excel files](https://gethue.com/blog/2021-11-15-create-sql-tables-from-execl-files/)
+- [Access your data in ABFS without any credential keys](https://gethue.com/blog/2021-09-21-access-your-data-in-abfs-without-any-credential-keys/)
+- [Create Phoenix tables in Just 2 steps](https://gethue.com/blog/2021-08-17-create-phoenix-tables-in-just-2-steps/)
+- [Open In Importer and Copy Path Options in Filebrowser](https://gethue.com/blog/2021-08-10-open-in-importer-and-copy-path-options-in-filebrowser/)
+- [Create SQL tables on the fly with zero clicks](https://gethue.com/blog/2021-07-26-create-sql-tables-on-the-fly-with-zero-clicks/)
+- [Azure Storage sharing by leveraging SAS tokens so that your users don’t need credentials](https://gethue.com/blog/2021-06-30-how-to-use-azure-storage-rest-api-with-shared-access-sginature-sas-tokens/)
+
+
+It has more than 650+ commits and 100+ bug fixes! For more details on all the changes, check out the [release notes](https://docs.gethue.com/releases/release-notes-4.11.0/).
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md:115]: This release of Hue has 650+ commits and 100+ bug fixes. For a complete list, see the [release notes](https://docs.gethue.com/releases/release-notes-4.11.0/).

**Review by pychheda — COMMENTED** [review]
I have completed my review.
Thanks for running this by me!

**Comment by agl29** [comment]
Thanks a lot @pychheda

**Review by pychheda — APPROVED** [review]
The blog looks good to be published.

**Pull request merged**